index.html

```diff
@@ -45,6 +46,47 @@ <h1 class="title_hiring"> We're hiring </h1>
     </div>
     <!-- здесь заканчивается Колина часть html -->
 
+    <!--Nazar part-->
+
+    <div class="page-content">
+        <div class="header-first"><span>Engage Visitors</span></div>
+        <div class="header-second"><span>Understand your audience</span></div>
+        <div class="description_content">
+          <p>
+            Early stage company? Eligible applicants get all of our products for
+            just $49 a month.
+          </p>
+        </div>
+        <div class="buttons">
+          <input class="blue page-buttons" type="button" value="Get Started" />  
+          <input class="white page-buttons" type="button" value="Learn More" />  
+        </div>
+        <div class="pics">
+          <div class="pictures">
+            <img class="pic" src="images/developer.png" alt="Developer" />
+            <br />
+            <span class="pic-text">Developers</span>
+          </div>
+          <div class="pictures">
+            <img class="pic" src="images/tv.png" alt="TV" />
+            <br />
```

Comment:
Don't use `br` tag
`.pictures` can be `display: flex;` and `flex-direction: column`